sszgen/testcases/time.go

```diff
@@ -0,0 +1,13 @@
+package testcases
+
+import "time"
+
```

Comment:
Add a `go:generate` expression like [this](https://github.com/ferranbt/fastssz/blob/master/sszgen/testcases/case1.go#L3) on the top to generate these test files automatically.